### PR TITLE
Billink 54 rebase

### PR DESCRIPTION
**Pull request by patrickcoppens**

### CODE REVIEW

File: view/frontend/web/template/payment/form.html [hunk 1]
```diff
@@ -20,49 +20,59 @@
...
-                                        autocomplete: 'off',
-                                        id: $parent.getCode() + '_customer_type_' + workflow.type.toLowerCase(),
-                                        title: $t('Customer Type ' + workflow.type)}"/>
-                        <label data-bind="attr: {for: $parent.getCode() + '_customer_type_' + workflow.type.toLowerCase()}"
-                               class="label">
-                            <span data-bind="text: workflow.type"></span>
-                        </label>
-                        <br/>
-                        <!--/ko-->
+                <div class="field NEW">
```
Comment (bakkerpj) [inline view/frontend/web/template/payment/form.html]: @patrickcoppens Can't find a reference to class `NEW`. Looks like a debug leftover?

File: view/frontend/web/template/payment/form.html [hunk 2]
```diff
@@ -139,47 +149,23 @@
...
-                               maxlength="100" name="payment[delivery_address_housenumber]"/>
-                    </div>
+
+                <div class="field" data-bind="visible: toggle">
+                    <button class="action action-update" type="button" data-bind="click: updateAddress, click: toggleView">
+                        <span data-bind="i18n: 'Update'">Update</span>
+                    </button>
                 </div>
+
                 <div class="field date"
```
Comment (bakkerpj) [inline view/frontend/web/template/payment/form.html]: @patrickcoppens In the above definitions the class "date" was removed. Shouldn't this one be removed as well?

**Review by bakkerpj — CHANGES_REQUESTED** [review]

File: view/frontend/web/template/payment/form.html (hunk 1 above)
Comment (patrickcoppens) [reply view/frontend/web/template/payment/form.html]: Done

File: view/frontend/web/template/payment/form.html (hunk 2 above)
Comment (patrickcoppens) [reply view/frontend/web/template/payment/form.html]: Done

**Review by bakkerpj — APPROVED** [review]

**Pull request merged**